README.md

```diff
@@ -19,17 +19,17 @@ You can subscribe to receive release updates by following these steps:
 ## Requirements
 
 * `minSdkVersion`: **21**
-* `compileSdkVersion`:**31**
-* Java 11 language feature support
+* `compileSdkVersion`:**33**
+* Java 11 language feature support (compile with Java 17 as of version 5.0.0)
 * [Android X](https://developer.android.com/jetpack/androidx/) as of version SDK 2.0.0
 * A Beacon Id created on [Help Scout](https://secure.helpscout.net/settings/beacons/)
 
 ### Supported platform and language versions
 
-* Android 11.0 to 13.0 (as noted above the "minSdkVersion: 21 (Android 5.0)" for greater compatiblity however we only offer support for issues on Android 11.0 to 13.0)
-* Kotlin 1.6
+* Android 11.0 to 13.0 (as noted above the "minSdkVersion: 21 (Android 5.0)" for greater compatiblity however we only offer support for issues on Android 11+)
+* Kotlin 1.8.20
```

Comment:
Great catch! I'm going to amend the Changelog because the [SDK is effectively using 1.8.20, not 1.8.21.](https://github.com/helpscout/beacon-android-sdk/blob/5710c71722a57a80915a4d5e810c5d8653b2a4fb/dependencies.gradle#L2)